metricbeat/metricbeat.yml

```diff
@@ -10,41 +10,13 @@
 #==========================  Modules configuration ============================
 metricbeat.modules:
```

Comment:
@monicasarbu This one is WIP. I will restore the old metricbeat.yml as soon as I added automated tests. This file should not change.